src/ply.rs

```diff
@@ -264,6 +264,19 @@ macro_rules! read_casted_property {
     )
 }
 
+// Similar, but creates a function that just advances the read pointer.
+macro_rules! create_skip_fn {
+    (&mut $size:ident, $num_bytes:expr) => (
```

Comment:
Not related to this macro but GitHub does not allow me to put the comment where it belongs. In `create_and_return_reading_fn` the macro takes a parameter `data_type` that is not mentioned in the macro body. Can it be removed or what am I missing?